.github/workflows/linting.yaml

```diff
@@ -0,0 +1,19 @@
+name: Linting workflows
+
+on:
+  push:
+    paths:
+      - .github/**
+
+permissions:
+  contents: none
+
+jobs:
+  actionlint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+      - name: "Check GitHub workflow files"
+        uses: docker://docker.mirror.hashicorp.services/rhysd/actionlint:latest
```

Comment:
do we need to do any pinning on docker images that we don't own? I'm uncertain here but we should validate.

Reply:
Is this image saved in our docker repository? Maybe someone has audited it before publishing?